fossunited/fossunited/doctype/foss_event_cfp_submission/foss_event_cfp_submission.py

```
@@ -372,3 +382,45 @@ def cfp_get_talk_video(self) -> str | None:
+
+        <p>Regards,<br>
+        FOSS United Team</p>
+        """
+
+        try:
+            frappe.sendmail(
+                recipients=to,
+                cc=emails,
+                subject=f"{self.full_name} has Withdrawn their proposal from {self.event_name}",
+                message=message,
+                reference_doctype=PROPOSAL,
```

Ruff (0.14.7)

Undefined name `PROPOSAL`

(F821)